lib/passageidentity/user_api.rb

```diff
@@ -240,23 +185,76 @@ def signout(user_id:)
 
     private
 
+    def create_v2(args: {})
+      warn '[DEPRECATED] the `create` method parameters will change to `args: CreateUserArgs`.' \
+      'Parameters will change on or after 2025-1.'
```

Comment:
this shouldn't have a deprecation message

Reply:
Copy paste errors.